flyteidl/protos/flyteidl/admin/agent.proto

```diff
@@ -256,3 +258,24 @@ message GetTaskLogsResponse {
     GetTaskLogsResponseBody body = 2;
   }
 }
+
+// Error message to propagate detailed errors from agent executions to the execution
+// engine.
+message AgentError {
+  // A simplified code for errors, so that we can provide a glossary of all possible errors.
+  string code = 1;
+  // A detailed error message.
+  string message = 2;
+
```

Comment:
We have to remove this, since `resource.Message` can bring message to us already.